mindsdb/integrations/mlflow_handler/mlflow/__about__.py

```diff
@@ -0,0 +1,9 @@
+__title__ = 'MindsDB MLflow connector'
+__package_name__ = 'mindsdb'
```

Comment:
Should we use `mindsdb` as the package name if this can be published as a separate package?